### ISSUE TITLE
College Algebra: style basic elements

### ISSUE THREAD

**Issue by wilkus27** [openstax/cnx-recipes#2444]
Add styles to:

- Links (https://github.com/openstax/cnx-recipes/issues/2144)
- Titles in Regular modules (https://github.com/openstax/cnx-recipes/issues/2144)
- Learning Objectives (https://github.com/openstax/cnx-recipes/issues/2190)
- Basic Table (https://github.com/openstax/cnx-recipes/issues/2204)
- Column Header Table (https://github.com/openstax/cnx-recipes/issues/2204)
- Unnumbered Unstyled Table (https://github.com/openstax/cnx-recipes/issues/2205)

**Comment by MartaSztuka** [comment]
Tested
![Screenshot from 2020-06-24 15-08-20](https://user-images.githubusercontent.com/47241114/85562256-9b631680-b62c-11ea-9e5f-4fcf8b37d74b.png)
![Screenshot from 2020-06-24 15-10-58](https://user-images.githubusercontent.com/47241114/85562700-044a8e80-b62d-11ea-8537-aa89213659cf.png)
![Screenshot from 2020-06-24 15-11-22](https://user-images.githubusercontent.com/47241114/85562708-06145200-b62d-11ea-960d-219beed74f41.png)
![Screenshot from 2020-06-24 15-13-15](https://user-images.githubusercontent.com/47241114/85563013-4a075700-b62d-11ea-8c98-db7c05650517.png)
![Screenshot from 2020-06-24 15-14-46](https://user-images.githubusercontent.com/47241114/85563202-77540500-b62d-11ea-9aef-fa9d8cc98d57.png)
![Screenshot from 2020-06-24 15-36-22](https://user-images.githubusercontent.com/47241114/85566622-7d97b080-b630-11ea-87ed-4d37ef2be56c.png)

**Comment by MartaSztuka** [comment]
Learning objectives `In this section students will:` isn't styled. @wilkus27 
![Screenshot from 2020-06-24 15-10-58](https://user-images.githubusercontent.com/47241114/85569491-df591a00-b632-11ea-8ec9-e06cc85240ad.png)

**Comment by wilkus27** [comment]
@lwtchu This is the same problem as here: https://github.com/openstax/content-managers/issues/1726 Do you want me to create new issue in content-managers repo for precalc books?

**Comment by lwtchu** [comment]
@wilkus27 i'll add it to the CM card

**Comment by lwtchu** [comment]
are y'all blocked on testing? where should i make the changes  cause i thought i made the change on katalyst01 but my memory could be failing me

**Comment by lwtchu** [comment]
@wilkus27 i'm adding some cases - i'll ping you when they're updated :)

**Comment by wilkus27** [comment]
@MartaSztuka Could you fetch new content and test Learning Objective in chapter which Larissa mentioned here: https://katalysteducation.slack.com/archives/CBW9DFKGF/p1593120723332100? :slightly_smiling_face:

**Comment by MartaSztuka** [comment]
Learning Objectives is styled in 1.1 Real Numbers: Algebra Essentials
![Screenshot from 2020-06-26 13-24-17](https://user-images.githubusercontent.com/47241114/85852319-a050d300-b7b0-11ea-9653-30c61118852a.png)
